src/extensions/renderer/canvas/drawing-edges.js

```diff
@@ -181,45 +181,42 @@ CRp.drawArrowheads = function( context, edge, drawOverlayInstead ){
 CRp.drawArrowhead = function( context, edge, prefix, x, y, angle ){
   if( isNaN( x ) || x == null || isNaN( y ) || y == null || isNaN( angle ) || angle == null ){ return; }
 
-  var self = this;
```

Comment:
OK.  It's probably just used in most functions in case a ref to `this` is needed in an inner function.  It may not be needed here, but a side benefit is smaller minified files.  `this` is always 4 chars minified because it's a keyword; `self` becomes something 1 char like `x`.